components/stepper/stories/template.js

```diff
@@ -96,3 +97,490 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const SizingTemplate = (args, context) => 
+	html` ${["s", "m", "l", "xl"].map((size) => {
+		return html`
+			<style>
+				.spectrum-Detail { 
+					display: inline-block;
+					/* Why seafoam? Because it separates it from the component styles. */
+					--mod-detail-font-color: var(--spectrum-seafoam-900);
+				}
+			</style>
+			<div style=${styleMap({
+				"display": "flex",
+				"flexDirection": "column",
+				"alignItems": "flex-start",
+				"gap": "8px",
+				"padding": "16px",
+			})}>
+				${Typography({
+					semantics: "detail",
+					size: "s",
+					content: [
+						{
+							s: "Small",
+							m: "Medium",
+							l: "Large",
+							xl: "Extra-large",
+						}[size],
+					],
+					customClasses: ["chromatic-ignore"],
+				})}
+				${Template({
+					...args,
+					size,
+				}, context)}
+			</div>
+		`;
+	})}`;
+
+export const DefaultTemplate = (args, context) => html`
```

Comment:
Nice job condensing these from separate examples on the docs page, for this template and the QuietTemplate.
One visual suggestion; you could put invalid in a second row (two rows of three). For example with `display: grid` and three `max-content` columns. I think that would make it easier to visually compare the two sets of steppers:
![Screenshot 2024-07-17 at 11 05 15 AM](https://github.com/user-attachments/assets/d81deb46-0b91-4c10-96b0-8681136c3615)

Reply:
oh yeah yeah yeah! Love that. https://github.com/adobe/spectrum-css/pull/2900/commits/ac78180e7a698f69aff5d1861f490ce86ace412e